.travis.yml

```diff
@@ -115,7 +115,7 @@ install:
 script:
 
 - echo 'Configuring Coq...' && echo -en 'travis_fold:start:coq.config\\r'
-- ./configure -local -usecamlp5 -native-compiler yes ${EXTRA_CONF}
+- ./configure -local -usecamlp5 -native-compiler yes -warn-error ${EXTRA_CONF}
```

Comment:
Please make the `-warn-error` setting match the default configuration. You could use `EXTRA_CONF` if you wanna create an optional allow fail target, but I suggest we tweak travis in a new PR, we could also cleanup the IDE building by using YAML refs.